qtop.py

```diff
@@ -1681,8 +1692,8 @@ def get_document(scheduling_system):
 
                 # justification for implementation:
                 # http://unix.stackexchange.com/questions/47407/cat-line-x-to-line-y-on-a-huge-file
-                line_offset = viewport.get_v_stop() - viewport.get_v_start()
-                cat_command = 'clear;tail -n+%s %s | head -n%s' % (viewport.get_v_start(), output_fp, line_offset)
+                # line_offset = viewport.v_stop - viewport.v_start
+                cat_command = 'clear;tail -n+%s %s | head -n%s' % (viewport.v_start, output_fp, viewport.get_v_term_size())
```

Comment:
could we, perhaps, pre-cache this stream of characters and then dump it in one go?
(I see some flicker when on slow screen sessions, eg. remote ssh)

OR, piping the output via `cat` or `tee`, might just help it somewhat!